### ISSUE TITLE
Automatically update Brew formula

### ISSUE THREAD

**Issue by alexander-schranz** [box-project/box#522]
Not sure but it looks for me the brew formula is outdated or gets an old version when running:

```bash
brew tap humbug/box
brew install box

# ==> Installing box from humbug/box
# ==> Downloading https://github.com/humbug/box/releases/download/3.8.4/box.phar
# Already downloaded: /Users/alexanderschranz/Library/Caches/Homebrew/downloads/ec5bbcd3526d2ea2e58d717a7852b2124707204078a1ddbf350abf733a5fa605--box.phar🍺  /usr/local/Cellar/box/3.8.4: 3 files, 1.8MB, built in 2 seconds
box --version
# Box version 3.8.4@120b0a3 2019-12-13 17:22:43 UTC
```

**Comment by alexander-schranz** [comment]
Seems like the version is hardcoded here: https://github.com/humbug/homebrew-box/pull/5

**Comment by theofidry** [comment]
Indeed. I was using a tool to try to keep it up to date but it was not working due to the hook being triggered too early (before the artefact was built) and was somehow removed when migrating to GitHub Actions

**Comment by alexander-schranz** [comment]
@theofidry maybe a github action could be used in sulu example have this github action in our main repo:

https://github.com/sulu/sulu/blob/master/.github/workflows/trigger-styleguide-build.yaml

Which will trigger a workflow in another repository:

https://github.com/sulu/sulu-javascript-docs/blob/master/.github/workflows/update-documentation.yaml#L14

Another way would be a github action cronjob (schedule) which maybe runs every day once to check if there is a new version and if it is it will push the updated formula.

**Comment by theofidry** [comment]
I would fine with either solution 👍

**Comment by alexander-schranz** [comment]
@theofidry Hope I will find some time soon to create a PR.

**Comment by theofidry** [comment]
Should be implemented via #832. Not sure yet that it works!
